File: Capacitaciones-Back/capacitaciones/src/main/java/com/example/capacitaciones/service/SeccionService.java
package com.example.capacitaciones.service;

import java.util.List;

import com.example.capacitaciones.DTOs.SeccionDTO;
import com.example.capacitaciones.model.Seccion;

public interface SeccionService {
    Seccion save(SeccionDTO seccionDTO);

    List<SeccionDTO> findAllDTOByIdCapacitacion(Long idCapacitacion);

    void delete(Long idSeccion);
}

File: Capacitaciones-Front/src/services/seccionService.js
import api from './http'

export const seccionService = {
  buscarSeccionesPorCapacitacion: (idCapacitacion) =>
    api.get(`/seccion/seccionPorCapacitacion/${idCapacitacion}`),
  save: (seccion) => api.post('/seccion', seccion),
  eliminar: (id) => api.delete(`/seccion/${id}`),
}

export default seccionService
